src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2107,6 +2107,26 @@
           "type": "string"
         },
 
+        "organization": {
+          "description": "The organization or company that produced the tool.",
+          "type": "string"
+        },
+
+        "product": {
+          "description": "A product suite to which the tool belongs.",
+          "type": "string"
+        },
+
+        "shortDescription": {
+          "description": "A brief, multiformat description of the tool.",
+          "$ref": "#/definitions/multiformatMessageString"
+        },
+
+        "longDescription": {
```

Comment:
>longDescription [](start = 9, length = 15)

Sorry, the issue was wrong. s/b `fullDescription` to match the corresponding property name in `reportingDescriptor`.

Reply:
good catch

---
In reply to: [261382346](https://github.com/Microsoft/sarif-sdk/pull/1315#discussion_r261382346) [](ancestors = 261382346)